z/zipkin/build_info.json

```diff
@@ -0,0 +1,22 @@
+{
+    "package_name": "zipkin",
+    "github_url": "https://github.com/openzipkin/zipkin",
+    "version": "zipkin-0.7.0",
+    "default_branch": "master",
+    "package_dir": "z/zipkin",
+    "build_script": "zipkin_ubi_8_7.sh",
+    "docker_build": false,
```

Comment:
This should be true, As we are adding docker build also.
Do other necessary changes also in build_info file